playbooks/roles/tower/defaults/main.yml

```diff
@@ -1,10 +1,11 @@
 ---
 # Ansible Tower Configurations
 tower_login: true
-tower_host: ''
-tower_username: ''
-tower_password: ''
-tower_verify_ssl: false
+tower_environment: "{{ lookup('env','tower_environment') | default('local') }}"
+tower_host: '{{ lookup("vars", "_".join((tower_environment, "tower_host")) ) }}'
+tower_username: '{{ lookup("vars", "_".join((tower_environment, "tower_username")) ) }}'
+tower_password: '{{ lookup("vars", "_".join((tower_environment, "tower_password")) ) }}'
+tower_verify_ssl: '{{ lookup("vars", "_".join((tower_environment, "tower_verify_ssl")) ) }}'
```

Comment:
> See here @StevenTobin [fheng/integreatly_dev#10](https://github.com/fheng/integreatly_dev/pull/10)

I'm using your integreatly_dev branch
```
  supporting_multiple_towers → dk ?:1  ~/git/integreatly_dev                                                    
❯ git remote -v
dk	git@github.com:davidkirwan/integreatly_dev.git (fetch)
dk	git@github.com:davidkirwan/integreatly_dev.git (push)
origin	git@github.com:fheng/integreatly_dev.git (fetch)
origin	git@github.com:fheng/integreatly_dev.git (push)
```

There's no dev_tower_verify_ssl var in that branch, and here you are specifically saying there should be one.

Reply:
Ahh oops! Cheers will fix this now

Reply:
Done